src/components/screens/home/balance-card.tsx

```diff
@@ -70,10 +70,13 @@ const BalanceCard = () => {
     <>
       <Card>
         <CardContent>
-          <h3 style={{ marginTop: 6 }}>{`${strings.timebank.balance} ${formattedBeforeDate}`}</h3>
+          <h3 style={{ marginTop: 6 }}>{`${strings.timebank.balance}`}</h3>
           <Grid container>
-            <Grid item xs={1}>
-              <ScheduleIcon />
+            <Grid item xs={12}>
+              {`${formattedBeforeDate}`}
```

Comment:
Good but remember to include the hint text as to what the date implies such as "Up until 'date' "